server/src/main/java/io/littlehorse/server/streamsimpl/lhinternalscan/publicrequests/ListNodeRuns.java

```diff
@@ -40,31 +37,17 @@ public GetableClassEnumPb getObjectType() {
         return GetableClassEnumPb.NODE_RUN;
     }
 
-    public InternalScan startInternalSearch(LHGlobalMetaStores stores) {
-        InternalScan out = new InternalScan();
-        out.storeName = ServerTopology.CORE_STORE;
-        out.resultType = ScanResultTypePb.OBJECT;
-        out.type = ScanBoundaryCase.BOUNDED_OBJECT_ID_SCAN;
-        out.partitionKey = wfRunId;
-        out.boundedObjectIdScan =
-            BoundedObjectIdScanPb
-                .newBuilder()
-                .setStartObjectId(wfRunId + "/")
-                .setEndObjectId(wfRunId + "/~")
-                .build();
-        return out;
-    }
-
     @Override
-    public TagStorageTypePb indexTypeForSearch() throws LHValidationError {
-        return null;
+    public TagStorageTypePb indexTypeForSearch(LHGlobalMetaStores stores)
+        throws LHValidationError {
+        return TagStorageTypePb.LOCAL;
     }
 
     @Override
     public void validate() throws LHValidationError {}
 
     @Override
     public SearchScanBoundaryStrategy getScanBoundary(String searchAttributeString) {
-        return null;
+        return new ObjectIdScanBoundaryStrategy(wfRunId);
```

Comment:
Quick question. I don't quite see where we tell the Scan Boundary Strategy that we are looking for `ScanResultType.OBJECT` here as opposed to `ScanResultType.OBJECT_ID`. Where does that happen?

Reply:
here:

https://github.com/littlehorse-enterprises/littlehorse/pull/12/files#diff-8e78027dbb3fb2fdad58b82d87d187b45f3d5c4c4f91e31defa0be3612b7d834R43